### DISCUSSION TITLE
Chasing smugglers is not fun.  Not fun at all.

### DISCUSSION THREAD

**Discussion by zjdev** [endless-sky/endless-sky#10451] [Balance, Mechanics, and Interactions]
I skipped ahead a little bit, modified my save file to trigger mission 3 and play through the Hai Reveal in 0.10.8.

I've gotten to the part where I'm looking for smuggler ships.  OK, easy enough to find them.  I can jump into a system and press "n" until I find a smuggler.  It takes about ten to fifteen minutes of real time to find one.  If I find one as I jump in, it is jumping out.  If I find one waiting over a planet, it enters the system, gets close enough for me to see it on screen, turns around and jumps out.

When I chase it, it is never in the system it headed to.  Start searching again - ten to fifteen minutes to get a glimpse of the ship.  At one point I was seeing a blackbird smuggler at fairly frequent intervals, probably three times in a ten minute period.  Each of the three times it was on the other side of the system and jumped out before I was even close to weapons range, much less boarding the dang thing.

I think that maybe they're scared of me, so I park my escorts and lurk cloaked.  Cloaking has the nice effect of making the Republic Intelligence ship drift away (I can even jump away from it entirely.)  But the smugglers don't let me get close, even cloaked.

**_3+ hours of real time into chasing smuggler ghosts, I'm wondering what the point is._**  The missions are entertaining (to me) because of the dialog and story progression.  A mindless jump, target to find nothing, jump, target to find nothing, jump, target and miss a smuggler because I'm pressing "n" to fast but it's gone by the time I cycle back through, jump, find a smuggler to watch it jump away, jump, target to find nothing, jump, target to find nothing, jump, target to find nothing, then land, take off again, target to find nothing is about as fun as reading this sentence a hundred times.

I do understand that the Hai Reveal storyline isn't finished, and that's the point of this comment.  If this is supposed to be a challenge, please figure out how to make it a 20 or 30 minute challenge before you go live with the story.  Not an hours long slog to eventually get to the point where I'm looking at the data files to figure out what I need to paste into a save file to just move the story along.

Thanks!

**Comment by TomGoodIdea** [comment]
Isn't this part of HR disabled?

**Reply by zjdev** [reply]
Yes.  "I skipped ahead a little bit, modified my save file to trigger mission 3 and play through the Hai Reveal in 0.10.8."

Which is why I made the suggestion. Since this is disabled, there's a chance to adjust this mission before it goes live.  Possibly as easy as changing the ship personalities.  It looks like they're all timid at the moment. (Or maybe just one smuggler in the north and one in the south to not be so shy.)

https://github.com/endless-sky/endless-sky/blob/master/data/hai/hai%20reveal%205%20philosophers.txt

**Comment by Quantumshark** [comment]
I recall having similar thoughts the last time I played these missions.

I'll make sure they're looked at properly, with a view to adjusting or, if necessary, replacing them when we get to working the rest of HR into the game.

**Reply by zjdev** [reply]
I played through again, and if endless-sky had difficulty levels these would work:
Hard:  leave the fleets at "timid"
Medium-hard: set the fleets to "timid target"  (you can see them on the mini-radar, which helps a lot.  But you still have seconds to react!)
Medium-easy: set the fleets to "timid target lingering" - they'll hang out  in the system a little longer.
Just let me get to the dialog already: set the fleets to "timid target staying" - OK, look for the ship flying in circles, disable & board. 

Medium-hard or medium-easy seems most workable.  I think medium-easy might be best for casual players like me who aren't otherwise gamers.  (I don't play FPS or combat games where a few nanoseconds of lag can get me killed, so I'm not lighting quick on the controls.)

IMHO target personality also fits the dialog well since you have a special scanner just for the smugglers.  (Especially since the pirate hunting missions highlight pirate ship targets that don't really have an in-game explanation of why they're highlighted.)

**Reply by TheGiraffe3** [reply]
I would assume the explanation is your ship identifies them as matching a job. And normally, they're coming right at you, so...
You might be interested in reading the Hai Reveal PR, MasterOfGrey made some good statements. #4952.
